scripts/bind.sh

```diff
@@ -22,7 +22,7 @@ while getopts "iawm" opt; do
       rm -rf ${BUILDDIR}/io.sonr.motor-sources.jar
       cd ${BUILDDIR}
       tar -czvf motor-${VERSION}-android.tar.gz io.sonr.motor.aar
-      rm -rf ${ANDROID_ARTIFACT}
```

Comment:
?

Reply:
This was deleting the .aar file which I need.